plugin-build/build.gradle.kts

```diff
@@ -65,6 +65,11 @@ dependencies {
 
     compileOnly("org.jetbrains.kotlin:kotlin-gradle-plugin:${KotlinCompilerVersion.VERSION}")
 
+    // compileOnly since we'll be shading the common dependency into the final jar
+    // but we still need to be able to compile it (this also excludes it from .pom)
+    compileOnly(Libs.SENTRY)
```

Comment:
I think this could be just `implementation` btw. I don't think there are gradle plugins that have Sentry on their classpath, and even if there are, we'll be just using whatever is the latest version of sentry.java, which is probably fine?

Reply:
Not sure, better safe than sorry here? Or do you see problems with shading?